scripts/setup.py

```diff
@@ -222,44 +221,6 @@ def main(args: Optional[List[str]] = None) -> None:
             'https://github.com/yarnpkg/yarn/releases/download/v%s/%s'
             % (common.YARN_VERSION, yarn_file_name), yarn_file_name)
 
-    # Adjust path to support the default Chrome locations for Unix, Windows and
-    # Mac OS.
-    if os.path.isfile('/usr/bin/google-chrome'):
-        # Unix.
-        chrome_bin = '/usr/bin/google-chrome'
-    elif os.path.isfile('/usr/bin/chromium-browser'):
-        # Unix.
-        chrome_bin = '/usr/bin/chromium-browser'
-    elif os.path.isfile('/usr/bin/brave'):
-        # Arch Linux.
-        chrome_bin = '/usr/bin/brave'
-    elif os.path.isfile('/usr/bin/chromium'):
-        # Arch Linux.
-        chrome_bin = '/usr/bin/chromium'
-    elif os.path.isfile(
-            '/c/Program Files (x86)/Google/Chrome/Application/chrome.exe'):
-        # Windows.
-        chrome_bin = (
-            '/c/Program Files (x86)/Google/Chrome/Application/chrome.exe')
-    elif os.path.isfile(
-            'c:\\Program Files (x86)\\Google\\Chrome\\Application\\Chrome.exe'):
-        chrome_bin = (
-            'c:\\Program Files (x86)\\Google\\Chrome\\Application\\Chrome.exe')
-    elif os.path.isfile(
-            '/mnt/c/Program Files (x86)/Google/Chrome/Application/chrome.exe'):
-        # WSL.
-        chrome_bin = (
-            '/mnt/c/Program Files (x86)/Google/Chrome/Application/chrome.exe')
-    elif os.path.isfile(
-            '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'):
-        # Mac OS.
-        chrome_bin = (
-            '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome')
-    else:
-        print('Chrome is not found, stopping ...')
-        raise Exception('Chrome not found.')
-
-    os.environ['CHROME_BIN'] = chrome_bin
```

Comment:
It looks like CHROME_BIN gets used by karma: https://unix.stackexchange.com/a/176986, so I'm not convinced that we can drop the CHROME_BIN environment variable. Did something change such that this is no longer required?

Reply:
Ah, good catch.